script/testing/artifact_stats/__main__.py

```diff
@@ -75,11 +76,17 @@ def collect_artifact_stats(collectors):
                         type=str,
                         help="Performance Storage Service password")
 
+    parser.add_argument("--github-token", 
+                        help="GitHub token for calling REST API")
+
     args = parser.parse_args()
 
     if args.debug:
         LOG.setLevel(logging.DEBUG)
 
+    if args.github_token:
+        os.environ['GITHUB_TOKEN'] = args.github_token
```

Comment:
I'm not a big fan of using an environment variable but it seemed better than passing it down through a bunch of functions as a parameter.